nservicebus/azure-service-bus/publisher-names-configuration.md

```diff
@@ -9,17 +9,17 @@ tags:
 reviewed: 2016-04-18
 ---
 
-When `EndpointOrientedTopology` is selected, a mapping between publisher names and message types has to be properly configured, in order to ensure that subscriber receives event messages.  
+When the [`EndpointOrientedTopology`](/nservicebus/azure-service-bus/topologies/#version-7-and-above-endpoint-oriented-topology) is selected, a mapping between publisher names and message types has to be properly configured, in order to ensure that subscriber can receive event messages.  
 
-The mapping can be configured for specific message type:
+The mapping can be configured for a specific message type:
 
 snippet: publisher_names_mapping_by_message_type
 
 
-or for an assembly containing message types:
+or for an assembly containing multiple message types:
 
 snippet: publisher_names_mapping_by_assembly
 
-In this case, the transport analyzes all types in the assembly to identify which are messages, according to the configured [`message conventions`](/nservicebus/messaging/conventions.md). Then for each message type, the transport registers a mapping between the type and the publisher name.
+In the latter case, the transport analyzes all types in the assembly to identify which are messages, according to the configured [`message conventions`](/nservicebus/messaging/conventions.md). Then for each message type, the transport registers a mapping between the type and the publisher name.
```

Comment:
Looks like the sentence is broken.

> In the latter case, the transport analyzes all types in the assembly to identify which are messages, according to the configured message conventions. Then for each message type, the transport registers a mapping between the type and the publisher name

The messages that are being looked at are solely events. 
How about this:

> In the latter case, the transport analyzes all types in the assembly to identify event messages. Then for each event message type, the transport registers a mapping between the type and the publisher name. Events are identified by NServiceBus marker interface `IEvent` or configured [`event convention`](/nservicebus/messaging/conventions.md).